man/systemd.service.xml

```diff
@@ -268,25 +270,33 @@
             anyway.</para></listitem>
           </itemizedlist>
 
-          <para>It is generally recommended to use <varname>Type=</varname><option>exec</option> for
-          long-running services whenever possible, as it is the simplest and fastest option. However, as this
-          service type won't propagate service start-up failures and doesn't allow ordering of other units
-          against completion of initialization of the service (which for example is useful if clients need to
-          connect to the service through some form of IPC, and the IPC channel is only established by the
-          service itself — in contrast to doing this ahead of time through socket or bus activation or
-          similar), it might not be sufficient for many cases. If so, <option>notify</option>,
-          <option>notify-reload</option>, or <option>dbus</option> (the latter only in case the service
-          provides a D-Bus interface) are the preferred options as they allow service program code to
-          precisely schedule when to consider the service started up successfully and when to proceed with
-          follow-up units. The <option>notify</option>/<option>notify-reload</option> service types require
-          explicit support in the service codebase (as <function>sd_notify()</function> or an equivalent API
-          needs to be invoked by the service at the appropriate time) — if it's not supported, then
-          <option>forking</option> is an alternative: it supports the traditional heavy-weight UNIX service
-          start-up protocol. Note that using any type other than
-          <option>simple</option>/<option>exec</option> possibly delays the boot process, as the service
-          manager needs to wait for service initialization to complete. (Also note it is generally not
-          recommended to use <option>idle</option> or <option>oneshot</option> for long-running services.)
-          </para>
+          <para>It is recommended to use <varname>Type=</varname><option>exec</option> for long-running
+          services, as it be generically useful and ensures process setup errors (e.g. errors such as a
```

Comment:
```suggestion
          services, as it is generically useful and ensures process setup errors (e.g. errors such as a
```

Reply:
I don't even know what "generically useful" is supposed to mean. Maybe just say "because it ensures that process setup errors …".